plonk/src/proof_system/verifier.rs

```diff
@@ -821,23 +820,6 @@ where
         self.domain.evaluate_vanishing_polynomial(*zeta)
     }
 
-    /// Evaluate the first and the last lagrange polynomial at point `zeta`
-    /// given the vanishing polynomial evaluation `vanish_eval`.
-    #[inline]
-    pub(crate) fn evaluate_lagrange_1_and_n(
```

Comment:
Since we are replacing this fn. Maybe it's better to have (an additional) `first_and_last_lagrange_coeffs`?

Reply:
see [70ef5f5](https://github.com/EspressoSystems/jellyfish/pull/611/commits/70ef5f545728e2ef95abf2443abf002a1814f84d)